index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey-From</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="title">freeCodeCamp Survey Form</h1>
+    <p class="description"><i>Thank you for taking the time to help us improve the platform</i></p>
+    
+    <div class="form-page">
+        <form id="survey-form" action="" method="post">
+        <div class="info-div">
+            <label for="name" id="name-label">Name</label><br>
+            <input class="input-class" type="text" name="" id="name" placeholder="Enter your name" required><br> 
+        </div>
+
+        <div class="info-div">
+            <label for="email" id="email-label">Email</label><br>
```

Comment:
please, i would suggest you Add comments to enable easy navigation through the code and which section is meant for a particular function